README.md

```diff
@@ -90,31 +81,31 @@ extension is recommended.
 
 ### How does this project relate to other tools?
 
-This project is forked from the
-[ReScript compiler](https://github.com/rescript-lang/rescript-compiler/),
-focused on a deeper integration with the OCaml ecosystem. This allows sharing
-code between backend and frontend using Dune's virtual libraries.
+This project is forked from an earlier version of the [ReScript
+compiler](https://github.com/rescript-lang/rescript-compiler/), focused on a
+deeper integration with the OCaml ecosystem. Such integration makes it easy to
+share code between backend and frontend using e.g. Dune's virtual libraries.
```

Comment:
Maybe adding a page under melange docs about sharing code seems like a great idea at this point

Reply:
cc @jchavarri

Reply:
Sure. Please open issue over there.